src/Schema/Schema.php

```diff
@@ -9,24 +9,23 @@
 use RalphJSmit\Laravel\SEO\Support\SEOData;
 use RalphJSmit\Laravel\SEO\Support\Tag;
 
+/**
+ * @deprecated Use CustomSchema paradigm
+ */
 abstract class Schema extends Tag
 {
     use Pipeable;
 
+    public string $tag = 'script';
+
     public array $attributes = [
         'type' => 'application/ld+json',
     ];
 
-    public string $context = 'https://schema.org/';
-
     public Collection $markup;
 
     public array $markupTransformers = [];
 
-    public string $tag = 'script';
-
-    public HtmlString $inner;
```

Comment:
`$inner` is now defined in Tag

Reply:
Not all tags had content, so hence it was only in the tags that are supposed to have content. But this is OK.